### DISCUSSION TITLE
Ability to create a sitemap relation for website.

### DISCUSSION THREAD

**Discussion by KrisKnez** [directus/directus#11342] [Feature Requests]
Currently, as far as I know, there is no way of create a tree structure for a specific collection.

I have a collection called page and I would like to organize it inside a tree structure so it looks something like this:
![image](https://user-images.githubusercontent.com/11775989/151717227-f860d0af-6f89-4c23-a66e-f9a2d7c6a500.png)

I have tried to implement this by creating a tree map inside the pages collection.
![image](https://user-images.githubusercontent.com/11775989/151717294-a3cfa5ad-8d9e-4a47-8a2c-f79afc6d8917.png)
Opening a specific page, you can see this field which looks quite nice:
![image](https://user-images.githubusercontent.com/11775989/151717325-5221854d-47af-43bd-8407-099d18cb7afb.png)

But it would be better if you could create a separate collection which would relate to the pages collection and have a tree map field for it like this:
![image](https://user-images.githubusercontent.com/11775989/151717359-ad42770e-372f-48d6-8075-8aa180f9b16b.png)

I wasn't able to implement this because apparently tree map only works recursively on a single collection.

Is there a way to implement this and if not would it be technically possible to make the tree map field work on another collection other than itself?

Related discussion: https://github.com/directus/directus/discussions/5546

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
